### ISSUE TITLE
discordjs should consider forking and maintaining https://github.com/discord/erlpack/

### ISSUE THREAD

**Issue by fredkilbourn** [discordjs/discord.js#6418]
**Is your feature request related to a problem? Please describe.**
https://github.com/discord/erlpack/ won't compile/install with node v16, which is required for discord.js v13.  The maintainers over there are really slow to respond on the verge of it being an abandoned project.  Users are trying to get a fix for 10 days now without even a response and the thread on that PR is devolving a bit: https://github.com/discord/erlpack/pull/40 .

**Describe the ideal solution**
discordjs should just fork and maintain this project, take the onus off of the users to fix this technical hurdle and guarantee more timely updates to match your own requirements.

**Describe alternatives you've considered**
I've considered installing a fork with the fixes applied, but it appears that because of the native bindings involved, discordjs won't recognize non-official builds/forks of https://github.com/discord/erlpack/.

**Additional context**
N/A

**Comment by Milo123459** [comment]
Best outcome, alternatively in client options providing an object with all erlpack functions for customisation

**Comment by Hazmi35** [comment]
I support this idea. Right now, I used my own fork [erlpack](https://github.com/hzmifork/erlpack), it's compatible with Node v16, plus the support of using erlpack with nodejs worker threads

**Comment by fredkilbourn** [comment]
Looks like we've gone through a number of alternate pull requests over at erlpack, current active one is now: https://github.com/discord/erlpack/pull/41

**Comment by fredkilbourn** [comment]
@iCrawl I agree with your sentiment about that crazy guy, but It's been 16 days on the current open PR without a peep from the maintainers there.  It seems really slow, but hopefully still forthcoming.  May follow your advice and drop erlpack altogether in my project because I've been roadblocked for almost a month waiting for this fix.

**Comment by Hazmi35** [comment]
> @iCrawl I agree with your sentiment about that crazy guy, but It's been 16 days on the current open PR without a peep from the maintainers there.  It seems really slow, but hopefully still forthcoming.  May follow your advice and drop erlpack altogether in my project because I've been roadblocked for almost a month waiting for this fix.

If you need to use it right now you could directly install from the fork while waiting for the PR to be merged.
